### DISCUSSION TITLE
Search Verticals - Grouping

### DISCUSSION THREAD

**Discussion by harishbalajigilead** [microsoft-search/pnp-modern-search#3003] [Ideas]
It would a good feature to group Search Vertical values into a container. For e.g., if I have bunch of Content sources for File Shares, I should be able to specify a group name called 'File Shares' when I configure Search verticals so that I can select the appropriate vertical from the group. 

The visual representation could be a vertical ellipsis. In this way, we can also avoid the verticals flowing to the 2nd or third line if we have more than 10 to 15 search verticals configured with lengthier names.

**Comment by wobba** [comment]
Adding an overflow with ... is probably the easier approach here, not having groups.
